src/pages/terms-and-conditions/_data-feed.js

```diff
@@ -48,7 +48,7 @@ export const DataFeed = () => (
             <li>
                 <TncListText>
                     {localize(
-                        'Please note that weekend quotes are ignored for the purpose of trade settlement. During weekends, the forex markets may occasionally generate prices; however, these prices are often artificial (traders sometimes take advantage of the illiquidity of the markets during weekends to push prices up or down). To avoid settling prices based on such artificial prices, it is Our policy not to count weekend prices towards trade settlement values (except for Volatility Indices, which are open during weekends).',
+                        'Please note that weekend quotes are ignored for the purpose of trade settlement. During weekends, the forex markets may occasionally generate prices; however, these prices are often artificial (traders sometimes take advantage of the illiquidity of the markets during weekends to push prices up or down). To avoid settling prices based on such artificial prices, it is Our policy not to count weekend prices towards trade settlement values (except for synthetic indices, which are open during weekends).',
```

Comment:
does it need to be capitalized `Synthetic Indices` similar to what it's replacing?

Reply:
I just copied. I think the QA team will leave a comment on the card if it is needed or better to check with the copywriter?

Reply:
Done.